model_compression_toolkit/core/pytorch/utils.py

```diff
@@ -38,11 +38,13 @@ def set_model(model: torch.nn.Module, train_mode: bool = False):
     model.to(device)
 
 
-def to_torch_tensor(tensor):
+def to_torch_tensor(tensor,
+                    torch_dtype=np.float32):
```

Comment:
Seems like it's `numpy_type`, no `torch_dtype`. Anyway, tou can just call it `dtype`